client/web/src/enterprise/batches/settings/CodeHostConnectionNode.tsx

```
@@ -31,12 +39,24 @@ export const CodeHostConnectionNode: React.FunctionComponent<React.PropsWithChil
     const ExternalServiceIcon = defaultExternalServices[node.externalServiceKind].icon
     const codeHostDisplayName = defaultExternalServices[node.externalServiceKind].defaultDisplayName
 
+    const [checkCred, { data: checkCredData, loading: checkCredLoading, error: checkCredError }] = useLazyQuery<
+        CheckBatchChangesCredentialResult,
+        CheckBatchChangesCredentialVariables
+    >(CHECK_BATCH_CHANGES_CREDENTIAL, {})
+
     const buttonReference = useRef<HTMLButtonElement | null>(null)
 
     const [openModal, setOpenModal] = useState<OpenModal | undefined>()
     const onClickAdd = useCallback(() => {
         setOpenModal('add')
     }, [])
+    const onClickCheck = useCallback<React.MouseEventHandler>(
+        async event => {
+            event.preventDefault()
+            await checkCred({ variables: { id: node?.credential?.id ?? '' } })
```

should we at least log the error here, so we don't swallow it entirely and make debugging hard?